plugins/chunter-resources/src/components/PinnedMessagesPopup.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+  import { ChunterMessage } from '@anticrm/chunter'
+  import contact, { Employee, EmployeeAccount, formatName } from '@anticrm/contact'
+  import { Ref } from '@anticrm/core'
+  import { Avatar, getClient, MessageViewer } from '@anticrm/presentation'
+  import { IconClose } from '@anticrm/ui'
+  import { createEventDispatcher } from 'svelte'
+  import { getTime } from '../utils'
+
+  export let pinnedMessages: ChunterMessage[]
+  export let employeeAcounts: EmployeeAccount[]
+
+  const client = getClient()
+  const dispatch = createEventDispatcher()
+
+  function getAvatar (id?: Ref<Employee>): string | null {
+    let avatar: string | null = null
+    client.findOne<Employee>(contact.class.Employee, { _id: id }).then((e) => (avatar = e?.avatar ?? null))
+    return avatar
```

Comment:
It always return null, because it's async func ``` async function getAvatar (_id?: Ref<Employee>): Promise<string | undefined> {
 if (_id === undefined) return
  return (await client.findOne(contact.class. Employee, { _id }))?.avatar```